dev-tools/docker/build_linux_test_image.sh

```diff
@@ -16,7 +16,7 @@
 
 ACCOUNT=droberts195
 REPOSITORY=ml-linux-test
-VERSION=1
+VERSION=3
```

Comment:
If I get this right,  with -e you can pass environment variables down:

```
docker build .... -e VERSION
```

and use it in `dev-tools/docker/linux_tester/Dockerfile` instead of requiring changing it there in sync with this.

Reply:
I didn't realise you could use a variable in the `FROM` clause in a `Dockerfile`, but you're right that for about a year it's been possible: moby/moby#31352

I don't think I should change how everything fits together in this PR though.  Any changes to how the different images are linked should be separate.